sdk/python/packages/flet/src/flet/cli/commands/build.py

```diff
@@ -36,42 +36,49 @@ def __init__(self, parser: argparse.ArgumentParser) -> None:
                 "status_text": "Windows app",
                 "output": "build/windows/x64/runner/Release/*",
                 "dist": "windows",
+                "can_be_run_on": ["Windows"],
             },
             "macos": {
                 "build_command": "macos",
                 "status_text": "macOS bundle",
                 "output": "build/macos/Build/Products/Release/{project_name}.app",
                 "dist": "macos",
+                "can_be_run_on": ["Darwin"],
             },
             "linux": {
                 "build_command": "linux",
                 "status_text": "app for Linux",
                 "output": "build/linux/{arch}/release/bundle/*",
                 "dist": "linux",
+                "can_be_run_on": ["Windows", "Linux"],
```

Comment:
You can't build for Linux on Windows, can't you?

Reply:
Yes... It was written as `Linux (wsl)` on the matrix. I didn't know what to do. I leave this code up to you 😶...